pkgs/development/tools/squawk/default.nix

```
@@ -0,0 +1,55 @@
+{ darwin
+, fetchFromGitHub
+, lib
+, libiconv
+, libpg_query13
+, openssl
+, pkg-config
+, rustPlatform
+, stdenv
+}:
+rustPlatform.buildRustPackage rec {
```

Since `libpg_query13` is only used by this package, I prefer to override it here:
```suggestion
let
  libpg_query13 = libpg_query.overrideAttrs (_: rec {
    version = "13-2.2.0";
    src = fetchFromGitHub {
      owner = "pganalyze";
      repo = "libpg_query";
      rev = version;
      hash = "sha256-gEkcv/j8ySUYmM9lx1hRF/SmuQMYVHwZAIYOaCQWAFs=";
    };
  });
in
rustPlatform.buildRustPackage rec {
```